R/arcadia.R

```diff
@@ -40,8 +40,8 @@ theme_arcadia <- function(x_axis_type = "numerical", y_axis_type = "numerical",
   y_axis_size <- if (y_axis_type == "categorical") 15 else 14.5
 
   # axis ticks
-  x_axis_ticks <- if (x_axis_type == "categorical") ggplot2::element_blank() else ggplot2::element_line(color="black", size = 0.5)
-  y_axis_ticks <- if (y_axis_type == "categorical") ggplot2::element_blank() else ggplot2::element_line(color="black", size = 0.5)
+  x_axis_ticks <- if (x_axis_type == "categorical") ggplot2::element_blank() else ggplot2::element_line(color="black", size = 0.35) # size ratio to pts is 2.13, so .75 in pts is .35 here
```

Comment:
So crazy that you had to use post-it math to figure this out! Thank you for doing that!